js/ui/endSessionDialog.js

```diff
@@ -80,6 +80,24 @@ const logoutDialogContent = {
     showOtherSessions: false,
 };
 
+
+const scheduledUpdatesDialogContent = {
+    subject: C_("title", "Power Off"),
+    subjectscheduledUpdates: ("title", "You have updates scheduled to happen"),
+    descriptionscheduledUpdates: function() {
+        return _("You need to leave the computer on to get scheduled updates");
+    },
+    showBatteryWarning: true,
+    confirmButtons: [{ signal: 'ContinueToShutdown',
+                       label:  C_("button", "Continue to shutdown") },
+                     { signal: 'changeSettings',
+                       label:  C_("button", "Change Settings...") },
+                     ],
+    iconName: 'system-shutdown-symbolic', /* FIXME: Use endless-auto-updates-scheduled-symbolic.svg */
```

Comment:
Hey @GeorgesStavracas 

Need a bit of help here. I couldn't figure out how to use ` endless-auto-updates-scheduled-symbolic.svg` as the icon here. Any ideas?

Reply:
I had problems in the past too with icons, and ultimately failed to figure out what was going on. Sorry, I don't think I have any valuable tip or comment to say :slightly_frowning_face: